src/set_theory/game/pgame.lean

```diff
@@ -613,21 +613,33 @@ by { cases x, refl }
 @[simp] theorem right_moves_neg (x : pgame) : (-x).right_moves = x.left_moves :=
 by { cases x, refl }
 
-/-- Turns a right move for `x` into a left move for `-x`. -/
-def to_left_moves_neg {x : pgame} (i : x.right_moves) : (-x).left_moves :=
-cast (left_moves_neg x).symm i
+/-- Turns a right move for `x` into a left move for `-x` and viceversa. -/
+def to_left_moves_neg {x : pgame} : x.right_moves ≃ (-x).left_moves :=
+equiv.cast (left_moves_neg x).symm
 
 /-- Turns a left move for `x` into a right move for `-x`. -/
-def to_right_moves_neg {x : pgame} (i : x.left_moves) : (-x).right_moves :=
-cast (right_moves_neg x).symm i
+def to_right_moves_neg {x : pgame} : x.left_moves ≃ (-x).right_moves :=
+equiv.cast (right_moves_neg x).symm
 
-/-- Turns a right move for `-x` into a left move for `x`. -/
-def neg_to_left_moves {x : pgame} (i : (-x).right_moves) : x.left_moves :=
-cast (right_moves_neg x) i
+/-- A left move of `-x` is a right move of `x`. -/
+theorem left_moves_neg_eq_right_moves {x : pgame} (i : (-x).left_moves) :
+  ∃ j : x.right_moves, i = to_left_moves_neg j :=
+by { cases x, exact ⟨i, rfl⟩ }
 
-/-- Turns a left move for `-x` into a right move for `x`. -/
-def neg_to_right_moves {x : pgame} (i : (-x).left_moves) : x.right_moves :=
-cast (left_moves_neg x) i
+/-- A right move of `-x` is a left move of `x`. -/
+theorem right_moves_neg_eq_left_moves {x : pgame} (i : (-x).right_moves) :
+  ∃ j : x.left_moves, i = to_right_moves_neg j :=
```

Comment:
This lemma is dumb given that `to_right_moves_neg` is an `equiv`.

Reply:
You're right! I actually managed to golf some more stuff thanks to that `equiv` you suggested.